fe2-android/app/src/main/java/com/github/dedis/student20_pop/qrcode/QRCodeScanningFragment.java

```diff
@@ -20,89 +22,114 @@
 import com.google.android.gms.vision.CameraSource;
 import com.google.android.gms.vision.barcode.BarcodeDetector;
 import java.io.IOException;
-
 /** Fragment handling the QR code scanning */
 public final class QRCodeScanningFragment extends Fragment {
-
   public static final String TAG = QRCodeScanningFragment.class.getSimpleName();
-
   private static final int HANDLE_GMS = 9001;
-
   private FragmentQrcodeBinding mQrCodeFragBinding;
   private QRCodeScanningViewModel mQRCodeScanningViewModel;
   private CameraSource camera;
   private CameraPreview mPreview;
   private BarcodeDetector barcodeDetector;
-
   /** Fragment constructor */
   public QRCodeScanningFragment(CameraSource camera, BarcodeDetector detector) {
     super();
     this.camera = camera;
     this.barcodeDetector = detector;
   }
-
   public static QRCodeScanningFragment newInstance(CameraSource camera, BarcodeDetector detector) {
     return new QRCodeScanningFragment(camera, detector);
   }
-
   @Override
   public void onAttach(@NonNull Context context) {
     super.onAttach(context);
   }
-
   @Override
   public View onCreateView(
-      @NonNull LayoutInflater inflater,
-      @Nullable ViewGroup container,
-      @Nullable Bundle savedInstanceState) {
-
+          @NonNull LayoutInflater inflater,
+          @Nullable ViewGroup container,
+          @Nullable Bundle savedInstanceState) {
     mQrCodeFragBinding = FragmentQrcodeBinding.inflate(inflater, container, false);
-
     FragmentActivity activity = getActivity();
     if (activity instanceof HomeActivity) {
       mQRCodeScanningViewModel = HomeActivity.obtainViewModel(activity);
-    } else {
+      mQrCodeFragBinding.addAttendeeTotalText.setVisibility(View.GONE);
+      mQrCodeFragBinding.addAttendeeNumberText.setVisibility(View.GONE);
+      mQrCodeFragBinding.addAttendeeConfirm.setVisibility(View.GONE);
```

Comment:
You can set the visibility to gone on the layout file and set it to visible only when you are scanning attendees